main.star

```diff
@@ -920,6 +920,11 @@ print(int(a+b), end="")
                 ),
             )
 
+            plan.run_sh(
```

Comment:
Maybe this should be after transfer? Not sure why launching a pytest may need a sleep.